docs/preview/features/logging.md

```diff
@@ -168,6 +168,8 @@ public class OrderController : ControllerBase
 }
 ```
 
+#### Excluding request/response bodies
```

Comment:
Do we need to update a TOC?

Reply:
No, as the TOC only contains the main headers.